SUASSystem/SUASSystem/sda_converter.py

```diff
@@ -141,7 +150,10 @@ def has_uav_completed_guided_path(self):
         """
         Returns True if the UAV has completed the guided path, False if not
         """
-        return self.current_path_index >= len(self.current_path)
+        if len(self.current_path) != 0:
+            return self.current_path_index >= len(self.current_path)
+        else:
+            return True
```

Comment:
In this case, you do not need the else statement. Simply put the ```return True``` after the if statement, and if the condition is not met, ```True``` will still always be returned.

Reply:
done